Open_Acidification_pH-stat_arduino/printSpecifiedLines.ino

```diff
@@ -1,20 +1,20 @@
 void printSpecifiedLines(String dirName, EthernetClient client, long startingLine, long numLines) {
   File myFile = SD.open(dirName);
   if (myFile) {
-    Serial.print("STARTINGLINE: ");
+    Serial.print(F("STARTINGLINE: "));
     Serial.println(startingLine);
-    Serial.print("NUMLINES: ");
+    Serial.print(F("NUMLINES: "));
     Serial.println(numLines);
     int printedLines = 0;
     // seek starting byte
     long startingByte = startingLine * (RECORD_LENGTH + 2);
     if (myFile.seek(startingByte)) {
-      Serial.println("SUCCESSFUL SEEK");
+      Serial.println(F("SUCCESSFUL SEEK"));
       printHeader(client, 200);
       client.println("time,tankid,temp,temp setpoint,pH,pH setpoint,onTime");
```

Comment:
I think you can also use F macro for `client.println` and `client.print`. Can you try it and see if it compiles?